TypeScript.py

```diff
@@ -47,6 +48,12 @@ def set_log_level(logger):
 
 TOOLTIP_SUPPORT = int(sublime.version()) >= 3072
 
+# Contains the template resource used to style signature overload popups
+popup_html = None
+
+# Contains the popup session when active
+popup_session = None
+
```

Comment:
It might be easier to refactor later to set "popup_html" and "popup_session" as static members of your command class, instead of a global variable.